src/Admin/Pool.php

```diff
@@ -136,9 +153,6 @@ public function __construct(
         $this->propertyAccessor = $propertyAccessor;
     }
 
-    /**
-     * @return array<string, array<string, AdminInterface>>
```

Comment:
Why did you remove the docs here?

Reply:
Psalm was failing and I'll check later today, but looks like this method does not work, even throws an error, I was going to check to deprecate it.

Based on this structure:

https://github.com/sonata-project/SonataAdminBundle/pull/6659/files#diff-96aac3a8c78cc9632a36cbb7489484b6c87ae336c1ceeaa028c4b265cafc6f38R41-R57

this method would try to to get and admin with id `label`, `label_catalogue`, `icon`, etc, that's why I guess it is not being used, but I haven't checked yet.